Fix typo. x.shape => dp.shape

yukinagae opened the pull request.

dustinvtran commented:
Can you actually rename the rv to be x? Not sure why it's dp.. must have been copypasta bug.

yukinagae commented:
Sure! I've just renamed the rv to be "x". Maybe copypasta from DirichletProcess ("dp") doc. I also found a similar copypasta in Empirical doc which may be bit confusing, so I fixed it as well.

I will create a new PR if you don't like the branch name "fix-typo-point-mass-doc" because I touched Empirical doc too.

dustinvtran commented:
This works. Thanks!
